spec/models/card_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Card do
+  let(:card) {Card.create(original_text: "hausE SOmE ДоМ", translated_text: "hauses")}
+  let(:card_bad) {Card.new(original_text: "home", translated_text: "home", review_date: Date.today)}
```

Comment:
Line is too long. [100/80]<br>Space missing inside {.<br>Space missing inside }.